9/DataTypes.md

```diff
@@ -295,16 +295,12 @@ DevPipeBlob is a Data Type to transfer data related to Pipes (see 7/Pipe).
 
  blob-data = *blob-data-element
  
- blob-data-element = name value inner-blob inner-blob-name 
-                     ; elements order implied by the above rule MAY be ignored    
-
- inner-blob = blob-data  
-
- inner-blob-name = name
+ blob-data-element = name value / blob-data 
```

Comment:
When looking at the doc, it seems like a name is missing in your new line.
This is why I've put:

`blob-data-element = name value / name blob-data `

But maybe the doc is wrong?
I'm still quite a newbie with Tango pipes.

Reply:
or maybe this would be also correct in abnf?:

`blob-data-element = name (value / blob-data) `

I think what you've written is equivalent to:
` blob-data-element = (name value) / blob-data `

Reply:
```suggestion
 blob-data-element = name (value / blob-data) 
```